modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js

```diff
@@ -82,15 +82,18 @@ export default ({section}) => {
 	}, [buildBreadcrumbNodesData, section]);
 
 	return (
+		
 		<section className="align-items-center d-flex">
 			<div className="questions-breadcrumb">
-				<ol className="breadcrumb mb-0 ml-2">
-					{breadcrumbNodes.length > MAX_SECTIONS_IN_BREADCRUMB ? (
-						<ShortenedBreadcrumb />
-					) : (
-						<AllBreadcrumb />
-					)}
-				</ol>
+				<div className="questions-container">
```

Comment:
this is very strange... a container that only contains the breadcrumb does not look logical. Can you use question-root to apply community styles?

Reply:
The tricky part is that the while the breadcrumb background should be aligned to `questions-root` (100% width) the `ol` tag (The contents of the breadcrumb) should be aligned to `questions-container` width. I can modify the styles of our custom css in liferay-community-theme to apply the same styles of `questions-container` to `breadcrumb` but that way, imho, the spec of "background should be aligned to root, and contents aligned to container" from the design wouldn't be enforced by the app.

Reply:
I've removed the questions-container and changed some of the bootstrap flexbox styles to fix the alignment later in custom portal theme

Reply:
But if you want to have `questions-container` values, you should wrap everything (and not only the breadcrumb and then close it) in it, right?